internal/version/version.go

```diff
@@ -20,7 +20,7 @@ import (
 )
 
 // Version is the version of the build.
-const Version = "1.27.0"
+const Version = "1.29.0-dev"
```

Comment:
we haven't had `-dev` in the version in a while, I am open to having it but what do other @cri-o/cri-o-maintainers think

Reply:
The dev was added as indicator for the deb/rpm package that we do not publish the final version. Not sure if it's still used anywhere else like the release notes.

Reply:
Hm, I guess we can also omit the dev :)